### ISSUE TITLE
Some doubts

### ISSUE THREAD

**Issue by yechaoying** [git-disl/awesome_LLM-harmful-fine-tuning-papers#2]
I don't think the formula you summarized in your paper is concise enough to understand.  In many papers,output = f(input) , loss = L(input) or loss = J(input,perturbation).…… I means "loss =  f_perturbation (weight) "  ->
![image](https://github.com/user-attachments/assets/bb7665a9-dc5c-471c-8950-cff428e1bb19)

**Comment by yechaoying** [comment]
alignment

**Comment by yechaoying** [comment]
Different empirical loss why not be used with one variable but different subscripts.😭

**Comment by huangtiansheng** [comment]
Hi,
Thanks for reaching out. In the paper we use f(w) to represent the safety alignment loss, h(w) represent the harmful loss. We don't use subscripts because it will make the formulation very untidy. In the area of  ML optimization, researchers are more getting used to the current formulation. For example, these very well-known papers use a similar notation:

ON THE CONVERGENCE OF FEDAVG ON NON-IID DATA   https://arxiv.org/pdf/1907.02189
FedProx https://proceedings.mlsys.org/paper_files/paper/2020/file/1f5fe83998a09396ebe6477d9475ba0c-Paper.pdf
Scaffold https://arxiv.org/pdf/1910.06378
